src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Tooltip/DefaultVSLSPTagHelperTooltipFactory.cs

```diff
@@ -11,6 +11,13 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Tooltip
 {
     internal class DefaultVSLSPTagHelperTooltipFactory : VSLSPTagHelperTooltipFactory
     {
+        // Internal for testing
+        internal static readonly VSImageElement TagHelperGlyph = new(
```

Comment:
I used the same glyph for all tag helper elements/attributes (the one that looks like `@<>`). If this isn't correct, let me know and I can update.